src/molecule/shell.py

```diff
@@ -88,14 +88,16 @@ def print_version(ctx, param, value):
 @click.option(
     "--base-config",
     "-c",
-    default=LOCAL_CONFIG,
+    multiple=True,
+    default="LOCAL_CONFIG",
```

Comment:
Is there a reason for changing the contents of the default value? I would expect that the default value should be `[LOCAL_CONFIG]` now, but to be honest, I do not know enough about click to be 100% sure.

Reply:
There are absolutely zero reason to change the content, it's a typo. Sorry about that